samples/client/petstore/rust-reqwest/src/apis/mod.rs

```diff
@@ -0,0 +1,32 @@
+use reqwest;
+use serde_json;
+
+#[derive(Debug)]
+pub enum Error {
+    Reqwest(reqwest::Error),
+    Serde(serde_json::Error),
+}
```

Comment:
It might be nice to use `error_chain` or `fail` instead of creating a custom error framework here. Non-essential, though, especially if this is what the hyper library already does.

Reply:
See https://github.com/rust-lang-nursery/failure or https://rust-lang-nursery.github.io/failure/.

In any case, what you've got is fine. Just something you might want to look at for future reference.